src/planning/plan/tests/planificator-test.ts

```diff
@@ -41,11 +39,6 @@ describe('planificator', () => {
             `Cannot construct key for '${storageKeyBase}' planificator storage key base`);
       assert(key.protocol.length > 0,
             `Invalid protocol in key for '${storageKeyBase}' planificator storage key base`);
-      if (!Flags.useNewStorageStack) {
-        const keyBase: KeyBase = key as KeyBase;
-        assert(keyBase.location && keyBase.location.length > 0,
-            `Invalid location in key for '${storageKeyBase}' planificator storage key base`);
-      }
     };
 
     verifySuggestion(storageKeyForTest(arc.id));
```

Comment:
do you want to remove the lines below:
verifySuggestion('firebase://arcs-test.firebaseio.com/123-456-7890-abcdef/1_2_3');
 verifySuggestion('pouchdb://local/testdb/');